cmd/dbmigrate/cmd/dbmigrate.go

```diff
@@ -98,10 +98,17 @@ Migration process:
 			if err != nil {
 				return fmt.Errorf("could not parse --%s option: %w", FlagBatchSize, err)
 			}
+
+			sourceDB, err := ReadDBBackendConfigValue(command)
```

Comment:
We should probably explain this is what is happening in the command Long section?  That it using the configured database and converting it to new source.   Not sure if we should also mention if the configured database is incorrect the process will not convert it correctly...thoughts?

Reply:
Agreed, that should be added.